uilib/defaults.py

```diff
@@ -23,7 +23,7 @@ def defaultMotor():
 def defaultPreferences():
     pref = preferences()
 
-    pref.general.props['burnoutWebThres'].setValue(0.03 / 39.37)
+    pref.general.props['burnoutWebThres'].setValue(0.01 / 39.37)
```

Comment:
A minor point, but we should use these default preferences in the motor sim class when preferences are not passed, instead of raw values.

Reply:
Hmm, I actually wasn't sure about the best way to proceed on this one. Defaults lives in uilib, but because simulation is in motorlib it would have to cross that barrier. I was hoping to keep the two separate, and I guess that means `runSimulation` shouldn't take in a preferences object at all. I think I will make it take in the four values it uses from preferences as separate arguments.